src/main/java/uk/gov/hmcts/reform/sendletter/controllers/reports/LetterListController.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sendletter.controllers.reports;
+
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sendletter.entity.BasicLetterInfo;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.model.out.LetterInfo;
+import uk.gov.hmcts.reform.sendletter.model.out.LettersInfoResponse;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.springframework.format.annotation.DateTimeFormat.ISO.DATE;
+
+@RestController
+public class LetterListController {
+
+    private final LetterRepository repository;
+
+    public LetterListController(LetterRepository repository) {
+        this.repository = repository;
+    }
+
+    @GetMapping(path = "/letters")
+    public LettersInfoResponse getLetters(@RequestParam(name = "date") @DateTimeFormat(iso = DATE) LocalDate date) {
```

Comment:
is it worth adding optional `service` param so that requester can filter them ?

Reply:
Also this would return bulk scan letter in prod used for smoke tests ?

Reply:
smoke test letters are filtered out here: https://github.com/hmcts/send-letter-service/pull/684/files

Reply:
>is it worth adding optional service param so that requester can filter them ?
It's not in the ticket, but good idea, let's see what business says after they see current solution